### PR TITLE
Cart page

### PR DESCRIPTION
**Pull request by OlhaKoziuk**
https://www.loom.com/share/0a31d68c25d34f05905c17fd82216e20?sid=c790fc3e-1766-464c-a57b-5e467a201529

### CODE REVIEW

**Review by mifyy — COMMENTED** [review]
it seems perfectly fine to me. The only thing I could spot is breadcrums instead of breadcrumbs :)

**Review by vlevik — APPROVED** [review]
good job! 👍

File: src/modules/CartPage/CartPage.tsx [hunk 1]
```diff
@@ -1,23 +1,94 @@
...
+        [styles.cart__DARK]: isDarkTheme,
+      })}
+    >
+      <div
+        className={cn(styles.breadcrums, {
+          [styles.contentDark]: isDarkTheme,
+        })}
+      >
+        Breadcrums
+      </div>
```
Comment (mpohorenyi) [inline src/modules/CartPage/CartPage.tsx]: This is not a breadcrumb, it's a regular back button with the same functionality as a browser button.
Use `./sec/modules/breadcrumbs`

File: src/modules/CartPage/CartPage.module.scss [hunk 2]
```diff
@@ -0,0 +1,126 @@
...
+  padding: 1rem 1.5rem 3.4rem;
+  background-color: $color__hover-and-bg;
+
+  @include onTablet {
+    padding: 1.5rem 2.5rem 4rem;
+  }
+
+  @include onDesktop {
+    padding: 2rem 2.5rem 5rem;
+  }
```
Comment (mpohorenyi) [inline src/modules/CartPage/CartPage.module.scss]: You cannot set external indents using BEM. In our project, we already have a container `./src/styles/block/_container.scss` applied to the main tag that envelops our entire routing

**Review by mpohorenyi — COMMENTED** [review]
Good job! 🔥
Please pay attention to the comments

File: src/modules/CartPage/CartPage.tsx [hunk 3]
```diff
@@ -27,13 +28,8 @@ export const CartPage: React.FC = () => {
...
     >
-      <div
-        className={cn(styles.breadcrums, {
-          [styles.contentDark]: isDarkTheme,
-        })}
-      >
-        Breadcrums
-      </div>
+      <Breadcrumbs />
+      {/* if I undestood your comment in a right way */}
```
Comment (mpohorenyi) [inline src/modules/CartPage/CartPage.tsx]: The BackButton component

**Review by mpohorenyi — APPROVED** [review]

**Pull request merged**